circus-train-core/src/main/java/com/hotels/bdp/circustrain/context/CommonBeans.java

```diff
@@ -105,6 +105,9 @@ private void configureMetastoreTunnel(MetastoreTunnel metastoreTunnel, Map<Strin
       properties.put(CircusTrainHiveConfVars.SSH_LOCALHOST.varname, metastoreTunnel.getLocalhost());
       properties.put(CircusTrainHiveConfVars.SSH_PRIVATE_KEYS.varname, metastoreTunnel.getPrivateKeys());
       properties.put(CircusTrainHiveConfVars.SSH_KNOWN_HOSTS.varname, metastoreTunnel.getKnownHosts());
+      properties.put(CircusTrainHiveConfVars.SSH_SESSION_TIMEOUT.varname, String.valueOf(metastoreTunnel.getTimeout()));
```

Comment:
These new features need to be mentioned in CHANGELOG

Reply:
yeah, forgot the CHANGELOG